src/level_four/core/tasks/send_transaction_tasks_solana.c

```diff
@@ -155,9 +150,9 @@ void send_transaction_tasks_solana() {
                                                 amount_string, amount_decimal_string, sizeof(amount_decimal_string)))
         break;
       instruction_scr_destructor();
-      snprintf(display, sizeof(display), UI_TEXT_VERIFY_AMOUNT, amount_decimal_string,
+      snprintf(display, sizeof(display), "%s\n%s", amount_decimal_string,
                get_coin_symbol(BYTE_ARRAY_TO_UINT32(var_send_transaction_data.transaction_metadata.coin_index), 0));
-      confirm_scr_init(display);
+      ui_scrollable_page(ui_heading_verify_amount, display, MENU_SCROLL_HORIZONTAL, false);
```

Comment:
```suggestion
      ui_scrollable_page(ui_heading_verify_transaction_fee, display, MENU_SCROLL_HORIZONTAL, false);
```

Reply:
The suggestion by @amanCypherock is valid this should be updated.